Hearthstone Deck Tracker/Controls/DeckImportingControl.xaml.cs

```diff
@@ -112,6 +112,31 @@ private void CheckBoxImportAll_OnModified(object sender, RoutedEventArgs e)
 			}
 		}
 
+		private void CheckBoxImport_OnChecked(object sender, RoutedEventArgs e)
+		{
+			int modifiedCount = 0;
+
+			if (ItemsControl == null)
+				return;
+
+			foreach(var item in ItemsControl.Items)
+			{
+				var c = (ContentPresenter)ItemsControl.ItemContainerGenerator.ContainerFromItem(item);
+				var chk = (CheckBox)c.ContentTemplate.FindName("CheckBoxImport", c);
+                if (chk.IsChecked.Value == true)
+					modifiedCount++;
+			}
+
+            if (modifiedCount == ItemsControl.Items.Count)
+                CheckBoxImportAll.IsChecked = true;
+
+		}
+
+        private void CheckBoxImport_OnUnchecked(object sender, RoutedEventArgs e)
+        {

```

Comment:
This can be converted to an expression. [See here](https://github.com/dotnet/roslyn/wiki/New-Language-Features-in-C%23-6#expression-bodies-on-method-like-members).